src/libs/merge/CMakeLists.txt

```diff
@@ -1,2 +1,30 @@
-file (GLOB SOURCES *.c)
-add_lib (merge SOURCES ${SOURCES})
+# Currently using LibGit2 gives memory errors on platforms other than Linux
```

Comment:
This is not a reason to completely disable it in other platforms (simply disable memory tests there, not the functionality)

Reply:
I disabled it because I was not sure if the problems could have a negative impact on the rest of Elektra.  

> simply disable memory tests there

We then have to disable the memory tests for all platforms, including Linux. I created #3141 to verify that I did this correctly.

Reply:
> We then have to disable the memory tests for all platforms, including Linux.

No, you can also exclude them for individual platforms. You can check in CMake which platform you are on.